src/test/regress/citus_tests/arbitrary_configs/citus_arbitrary_configs.py

```diff
@@ -128,13 +128,26 @@ def copy_test_files(config):
                 colon_index = line.index(":")
                 line = line[colon_index + 1 :].strip()
                 test_names = line.split(" ")
-                copy_test_files_with_names(test_names, sql_dir_path, expected_dir_path)
+                copy_test_files_with_names(test_names, sql_dir_path, expected_dir_path, config)
 
 
-def copy_test_files_with_names(test_names, sql_dir_path, expected_dir_path):
+def copy_test_files_with_names(test_names, sql_dir_path, expected_dir_path, config):
     for test_name in test_names:
+        # make empty files for the skipped tests
+        if test_name in config.skip_tests:
+            expected_sql_file = os.path.join(sql_dir_path, test_name + ".sql")
+            with open(expected_sql_file, 'w') as f:
+                f.write("")
```

Comment:
Do we need the `write` function here? We actually don't write anything.
Not sure but `open` with mode "a" could simply do what we need

Reply:
I can actually probably reduce these likes to `open(expected_sql_file, 'x').close()`.